pubspec.yaml

```
@@ -1,7 +1,7 @@
 name: built_html
 version: 0.1.3
```

```suggestion
version: 0.1.4
```

And then add a changelog entry please :)